stingray/io.py

```diff
@@ -680,7 +687,7 @@ def _save_ascii_object(object, filename):
 def _retrieve_ascii_object(object, filename):
     pass
 
-def write(input, filename, format = 'pickle'):
+def write(input, filename, format = 'pickle', save_as_dict = True):
```

Comment:
I'd prefer the `**kwargs` way to go. It's obviously going to be messy with lots of helper arguments for individual formats.